imagestreams/nodejs-centos.json

```diff
@@ -0,0 +1,147 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "nodejs",
+    "annotations": {
+      "openshift.io/display-name": "Node.js"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "Node.js (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js applications on UBI. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/12/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major version updates.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "supports":"nodejs",
+          "sampleRepo": "https://github.com/sclorg/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "12-ubi8"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "12-ubi8",
+        "annotations": {
+          "openshift.io/display-name": "Node.js 12 (UBI 8)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js 12 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/12/README.md.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "version": "12",
+          "sampleRepo": "https://github.com/sclorg/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "registry.redhat.io/ubi8/nodejs-12:latest"
```

Comment:
So an OKD vs. OCP thought @yselkowitz  .... when you take the next step and start updating openshift/library with this stuff, was your thought that the community.yaml would be updated to point to this file, the official.yaml, or both?

I ask because @vrutkovs recently updated samples operator to allow for installing a set of samples, and I'm pretty sure one of the stipulations was that none of the content would come from `registry.redhat.io` since that requires credentials, and in particular, credentials which are not needed as part of the install pull secret for installing OKD (correct me if I'm wrong @vrutkovs ).  

Whereas with OCP the registry.redhat.io/TBR creds are suppose to be in the install pull secret.

So if all that lines up, then we cannot include `12-ubi8`  in the community/OKD version.  Only in the official/OCP version.

Reply:
+1, community imagestreams should not have images from `registry.redhat.io` as those require a pull secret.

It seems there is `registry.access.redhat.com/ubi8/nodejs-12:latest` - could we use that?

Reply:
That's good point, for the community variant of the imagestreams, we should refer to `registry.access.redhat.com/ubi8/...`. All the UBI images should be available in this non-authenticated registry as well as in the authenticated registry `registry.redhat.io`.

Reply:
Ok, fixed that locally, will repush after a resolution to the ImageStreamTag comment.